extensions/core/interfaces/code/interface.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <div class="interface-code">
+    <codemirror ref="cm"
```

Comment:
I personally prefer to move the first attribute to a new line and indent the following ones on the same level, that way we prevent a big indent "gap":

```vue
<!-- Before -->
<codemirror ref="cm"
            :value="options.placeholder"
            :options="cmOptions"
            @ready="onCmReady"
            @focus="onCmFocus"
            @input="onCmCodeChange"
></codemirror>

<!-- After -->
<codemirror
  ref="cm"
  :value="options.placeholder"
  :options="cmOptions"
  @ready="onCmReady"
  @focus="onCmFocus"
  @input="onCmCodeChange">
</codemirror>
```